src/app/app.component.ts

```diff
@@ -9,4 +9,10 @@ import { Component } from '@angular/core';
 export class AppComponent {
   title = 'SplitWise';
   constructor(public router : Router){}
+
+  dropdownMenus = ['Profile','Logout'];
+
+  isCurrentUrl(url: string): boolean {
+    return this.router.url === url;
+  }
```

Comment:
In the function isCurrentUrl(), use angular router events to detect the current url don't pass url string from html which you have learned in angular router